lib/views/loading_indicator/loading_indicator.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
+
+class LoadingIndicator extends StatelessWidget {
+  const LoadingIndicator({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const SizedBox(
+      width: 80,
```

Comment:
@nmint8m Sorry, but I did not see the update on it. What I mean is instead of hard coding with and height `80`, we can create a `width` and `height` as params, which have a default value is `80`. By using this way, we can resize this indicator in any widget with any size we want.